### PR TITLE
Demo

### PR DESCRIPTION
**Pull request by heurezjusz**
Demo script created, still needs documentation

### CODE REVIEW

**Comment by marcinp7** [comment]
Change the name 'demo.py' to something more specific.

File: demo/demo.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import argparse
+from argparse import RawTextHelpFormatter
+from copy import deepcopy
```
Comment (marcinp7) [inline demo/demo.py]: Unused import

**Comment by marcinp7** [comment]
Add a comment at the beggining of the file to briefly summarize what that demo is doing.
I think it would be a good idea to set displaying help as the default behavior. If not, then add information about default configuration and -h option in the comment.

File: demo/config.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        net.data_loader = MNISTDataLoader()
+        return net
+    raise NotImplementedError
+
+
+def ok():
+    """
+        prints a good looking "OK" on the stdout.
+    """
+    print "[ \033[32mOK\033[39m ]"
```
Comment (marcinp7) [inline demo/config.py]: No newline at the end of file

**Comment by marcinp7** [comment]
Splitting the demo into 2 files might be confusing for users, because it causes that not all files in the 'demo' directory are indeed demos to be run. I would keep whole demo in one file or move 'config.py' to some subfolder of 'demo' (and maybe change the name from 'config.py' to something less generic)

File: demo/demo.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+                         "Amount of datasets depends on algorithm:\n"
+                         " * simple_neuron_deleter (sender): 2\n"
+                         " * simple_neuron_deleter2 (sender2): 2\n"
+                         " * sparsify_smallest_on_network (rat): 3\n",
+                    default=0)
+
+
+args = parser.parse_args()
+
+dataset = datasets[args.algorithm][args.dataset]
```
Comment (marcinp7) [inline demo/demo.py]: Maybe some error handling for incorrect -d value? Right now it gives error message that is not so helpful for user.

Comment (heurezjusz) [inline demo/demo.py]: Good one

**Comment by heurezjusz** [comment]
Done

**Pull request merged**